wca-guidelines.md

```diff
@@ -53,7 +53,7 @@ To be more informative, each Guideline is classified using one of the following
 - 2i++) [ADDITION] The organization team may provide the competitor an unofficial stopwatch for viewing the elapsed time (started together with the main stopwatch), in which case the competitor is not permitted to touch the official stopwatch.
 - 2i+++) [REMINDER] Bluetooth puzzles are considered electronic devices.
 - 2i1b+) [CLARIFICATION] This includes relevant devices which are switched off or disconnected.
-- 2i1c+) [CLARIFICATION] Electronic hand warmers may be used before or after an attempt only. Non-electronic hand warmers may be used at any time during an attempt.
+- 2i1c+) [CLARIFICATION] Electronic hand warmers are still considered electronic devices. Non-electronic hand warmers may be used at any time during an attempt.
```

Comment:
Given that people may refer to this without the corresponding Regulation (2i) as context, I'd suggest something more like this.

```suggestion
- 2i1c+) [CLARIFICATION] Electronic hand warmers are considered electronic devices, and therefore are not permitted while inspecting or solving. However, non-electronic hand warmers may be used at any time during an attempt.
```

This duplicates the "inspecting or solving" part of 2i, but I think that's worth it for clarity.

Reply:
LGTM.

Reply:
It's a nice wording in my opinion!

Reply:
LGTM also!